tools/scripts/ManifestBuilder.rb

```diff
@@ -157,18 +161,6 @@ def download_entire_animation_library
         #{dim 'd[ o_0 ]b'}
     EOS
 
-    if @options[:spritelab] && @options[:upload_to_s3]
```

Comment:
Thanks for removing this! Looks like it might've been a merge conflict on my part 😅